server/controllers/authController.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-unused-vars */
+import { Request, Response } from 'express';
+import { sign } from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import loginValidation from '../utils/validation/loginValidation';
+import User from '../database/Models/User';
+import { CustomError } from '../utils';
+
+const login = async (req: Request, res: Response) => {
+  const { email, password }: {email: string, password: string} = req.body;
+
+  await loginValidation(req.body);
+
+  const user = await User.findOne({ email });
+
+  if (!user) {
+    throw new CustomError('User not found', 404);
+  }
+
+  const validPassword: boolean = await bcrypt.compare(password, user.password);
+
+  if (!validPassword) {
+    throw new CustomError('Invalid password', 400);
+  }
+
+  if (!user.is_confirmed) {
+    throw new CustomError('Please Verify your email', 401);
+  }
+
+  const token: string = sign({ id: user.id }, process.env.JWT_SECRET ?? '', { expiresIn: '1h' });
```

Comment:
You are using sign sync, so convert it to promise in the utils and use it here.
And restructure all your variables (JWT_SECRET, id, and is_confirmed)